package.json

```diff
@@ -29,6 +29,7 @@
     "http-status-codes": "^2.3.0",
     "keycloak-js": "^24.0.4",
     "launchdarkly-vue-client-sdk": "^2.2.0",
+    "lodash": "^4.17.21",
```

Comment:
Would be good to avoid importing the whole library if possible. Since you are only using isEqual and omit I think you can import those specifically